styles/main.css

```diff
@@ -1,33 +1,20 @@
 body {
 	background-color: #D09B61;
 	font-family: 'Open Sans', sans-serif;
-	font-family: 'Roboto', sans-serif;
-	font-family: 'Lato', sans-serif;
-	font-family: 'Montserrat', sans-serif;
-	font-family: 'Raleway', sans-serif;

```

Comment:
Just use one font family property.  If you are trying to use these other fonts as fallbacks you should use a rule like `font-family: 'Open Sans', 'Roboto', 'Lato', 'Montserrat', 'Raleway', sans-serif;`